packages/x-date-pickers/src/PickersActionBar/PickersActionBar.tsx

```diff
@@ -38,9 +35,11 @@ const PickersActionBarRoot = styled(DialogActions, {
  * - [PickersActionBar API](https://mui.com/x/api/date-pickers/pickers-action-bar/)
  */
 function PickersActionBar(props: PickersActionBarProps) {
-  const { onAccept, onClear, onCancel, onSetToday, actions, ...other } = props;
+  const { actions, ...other } = props;
```

Comment:
we should not destruct this, as the `actions` are only used once.

```suggestion
```

Instead use `if (props.actions == null || props.actions.length === 0) {` (L44) and `props.actions.map` (L48)

Reply:
But if we do so, we forward `actions` to `PickersActionBarRoot`, which is problematic

And in the vast majority of cases, `actions` is not empty, so moving the spread below the `if(actions == null || actions.length === 0` does not bring a lot of value IMHO.